### DISCUSSION TITLE
Is there a way to disable images upload?

### DISCUSSION THREAD

**Discussion by wblondel** [umputun/remark42#930] [Q&A]
Hello,

I don't want my visitors to upload images, as I am responsible for what I store on my server and I don't trust my visitors ( 😄 ).
Is there a way to disable this feature ?

**Comment by umputun** [comment]
We don't have a nice and direct way to disable images. However, by setting IMAGE_MAX_SIZE  (or --image.max-size) to something low, for instance `1`, you effectively will reject the upload of any image.

Pls note - because this is not the real "disable image upload" flag, but rather a workaround, UI won't know anything about it and users will able to drag-drop images into a comment box. However, those images won't be saved or previewed and an attempt to publish such a comment will give an error, smth like this:

<img width="752" alt="ny6ns-202103-27141534-b1bvk" src="https://user-images.githubusercontent.com/535880/112732270-f18c1f80-8f06-11eb-9d86-7d4ecd526807.png">
 
One more thing - anonymous (truly untrusted users) can't upload images
